### ISSUE TITLE
cwl workflow

### ISSUE THREAD

**Issue by NiklasTR** [labdao/plex#194]
run in CWL outside of plex and report findings

**Comment by NiklasTR** [comment]
exploring LangChain for linking tasks over PLEX

**Comment by NiklasTR** [comment]
creating a dedicated branch with the prototype-CWL implementation as we are gradually transitioning

**Comment by mr-c** [comment]
Hello Plex folks; we'd love to hear more about Plex/LabDAO at an upcoming CWL user group meeting. Would someone like to volunteer to present? 30 minutes or less, casual atmosphere

https://www.commonwl.org/community/#when

**Comment by NiklasTR** [comment]
Hi Michael - thank you for the invitation! - we'd love to join. Next meeting -on September 18th- would work for me.

**Comment by mr-c** [comment]
> Hi Michael - thank you for the invitation! - we'd love to join. Next meeting -on September 18th- would work for me. Is that a good time on your side?

Glad to hear it! We've got a speaker already for the 18th (AWS HealthOmics & their CWL support); How about October 2nd?

**Comment by NiklasTR** [comment]
October 2nd sounds great as well - count us in! Just to confirm: October 2nd, 5PM Berlin time - at the link provided on [the website](https://arewemeetingyet.com/New%20York/2022-10-03/11:00/w/Weekly%20CWL%20Video%20Chat#eyJ1cmwiOiJodHRwczovL3d3dy5jb21tb253bC5vcmcvY29tbXVuaXR5LyNjd2wtb25saW5lLWNvbW11bml0eS1tZWV0aW5ncyJ9)?

**Comment by mr-c** [comment]
> October 2nd sounds great as well - count us in! Just to confirm: October 2nd, 5PM Berlin time - at the link provided on [the website](https://arewemeetingyet.com/New%20York/2022-10-03/11:00/w/Weekly%20CWL%20Video%20Chat#eyJ1cmwiOiJodHRwczovL3d3dy5jb21tb253bC5vcmcvY29tbXVuaXR5LyNjd2wtb25saW5lLWNvbW11bml0eS1tZWV0aW5ncyJ9)?

That's correct! We meet on Matrix, if you haven't used that before then https://www.commonwl.org/community/#cwl-online-community-meetings gives quick instructions
